dashboard/app/controllers/pd/professional_learning_landing_controller.rb

```diff
@@ -4,11 +4,18 @@ class Pd::ProfessionalLearningLandingController < ApplicationController
   def index
     # Get the courses that this user teaches
     workshops = Pd::Workshop.enrolled_in_by(current_user)
```

Comment:
Note this is counting enrollments, not attendance. This might be ok, but I just wanted to point it out. This will include teachers who registered, but did not show up for, a workshop.